cmd/root.go

```diff
@@ -97,18 +100,30 @@ func startGohaqd(cmd *cobra.Command, args []string) {
 		QueueUrl:        q.QueueUrl,
 		WaitTimeSeconds: aws.Int64(20),
 	}
+	sem = make(chan *sqs.Message)
```

Comment:
@ApsOps This channel will have only one message at a time?
Instead of this approach I think we should go with buffered channel.
`sem = make(chan *sqs.Message, parallelRequests)`